File: Week2/day02/draw-diamond.py
# Write a program that reads a number from the standard input, then draws a
# diamond like this:
#
#
#    *
#   ***
#  *****
# *******
#  *****
#   ***
#    *
#

num = int(input('My man, gimme a number: '))
for j, i in zip(range(2 * num - 1 , 0, -2), range(0, 2 * num, 2)):
    print(' ' * (i // 2) + '*' * j)
 list =

File: Week7/day02/draw-diamond.js
'use strict';

var lineCount = 7;



// Write a program that draws a
// diamond like this:
//
//
//    *
//   ***
//  *****
// *******
//  *****
//   ***
//    *
//
// The diamond should have as many lines as lineCount is

for (var i = 1, j = 4; i < lineCount * 2 + 1, 0 < j; i += 2, j--) {{
        console.log(' '.repeat(j) + '*'.repeat(i));
    }
}
for (var i = 2, j = 5; i < lineCount * 2 + 1, 0 < j; i++, j-=2) {{
        console.log(' '.repeat(i) + '*'.repeat(j));
    }
}
